src/foam/nanos/controller/ApplicationController.js

```diff
@@ -377,6 +379,17 @@ foam.CLASS({
       });
     },
 
+    function requestCapability(capabilityInfo) {
+      var self = this;
+      return new Promise(function(resolve, reject) {
```

Comment:
When does `resolve` get called? Don't we need something like `loginSuccess` for that?

Reply:
Right now it doesn't. I'm thinking `resolve` and `reject` should be passed to CapabilityInterceptView. If you agree then I'll add the properties to CapabilityInterceptView in this PR

Reply:
That could work.